install_config/aggregate_logging.adoc

```diff
@@ -1273,16 +1273,6 @@ logs, use the name `.operations` as the project name.
 |Use `.defaults` as the `$PROJECT_NAME` to set the defaults for projects that are
 not specified.
 
-|`runhour`
```

Comment:
@josefkarasek 
Should we delete runhour, runminute, and timezone variables from doc?

Reply:
@juzhao you're right. We should keep `runhour` and `runminute` as they allow users to specify at what part of the day the job will be scheduled.

The `timezone` timezone variable is no longer configurable as this value is sourced from Openshift master nodes.